app/models/concerns/orchestration/dhcp.rb

```diff
@@ -173,11 +173,9 @@ def dhcp_update_required?
       (!old.build? && build? && !all_dhcp_records_valid?))
     # Handle jumpstart
     # TODO, abstract this way once interfaces are fully used
```

Comment:
I wonder if this TODO isn't actually solved with 4cbf879ed5957978afd74b3707c8a6b16fa18225. Probably not for this PR, but something to investigate.

Reply:
I tried doing something here https://github.com/theforeman/foreman/pull/10261